src/hdmf/container.py

```diff
@@ -749,10 +755,58 @@ def _generate_field_html(self, key, value, level, access_code):
 
         return html_repr
 
+
     def _generate_array_html(self, array, level):
         """Generates HTML for a NumPy array."""
-        str_ = str(array).replace("\n", "</br>")
-        return f'<div style="margin-left: {level * 20}px;" class="container-fields">{str_}</div>'
+
+        # This is a placeholder function, you should define your own conversion
```

Comment:
Could you clarify this part, when/where should you define your own conversion?

Reply:
Outdated comments I wll remove.